app/styles/HeaderComp.scss

```diff
@@ -0,0 +1,97 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+}
```

Comment:
reset css should exist in some common or global css file

Reply:
Done.